components/support/sync-telemetry/metrics.yaml

```diff
@@ -443,3 +443,50 @@ bookmarks_sync:
       - sync-core@mozilla.com
     expires: never
     lifetime: ping
+fxa_tab:
+  sent:
+    type: event
+    description: >
+      Recorded when a tab is sent.
+    send_in_pings:
+      - sync
+    extra_keys:
+      flow_id:
+        description: >
+          A guid, unique for the URL being sent but common for all target
+          devices.
```

Comment:
Looking at other documentation and the Desktop data review, this should be fine, but could you clarify here?

Does "unique guid for the url being sent" imply that this is some kind of one-way hash from the url? Is its uniqueness is scoped to the Firefox account, and this `flow_id` would not be the same for other clients?

Mainly I want to verify something like "the ID is a randomly generated GUID so can not identify a **user or tab**" from
https://bug1639781.bmoattachments.org/attachment.cgi?id=9157561
(but that's from stream id)

Since this is also documented in the main firefox documentation, would recommend linking to this doc here as well:
https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/data/sync-ping.html

Reply:
The GUID is randomly generated and unrelated to the tab or user - I've updated the descriptions to reflect that and also add the link back to desktop.